### PR TITLE
Rework getting started and workflow

### PR DESCRIPTION
**Pull request by ferd**
One thing I'm noticing is that we'll have to change all the `doc:<chapter>` links to `/docs/<chapter>` apparently, so I'll make a PR reworking more links at some point.

### CODE REVIEW

File: content/en/docs/getting-started.md [hunk 1]
```diff
@@ -4,26 +4,30 @@ weight: 1
 excerpt: ""
 ---
 
+Rebar3 is the standard build tool within the Erlang community. It essentially bundles many of the other tools shipping with Erlang along with a few open-source ones, and makes them all work under a unified project structure.
```
Comment (tsloughter) [inline content/en/docs/getting-started.md:7]: "essentially bundles" -- it doesn't bundle the OTP tools, maybe some better wording for how it bundles the use of those tools but not the tools themselves?

File: content/en/docs/workflow.md [hunk 2]
```diff
@@ -1,49 +1,41 @@
...
-|A full, self-contained, executable system|release or umbrella|This is the recommended production deploy for Erlang systems.
-
-See the [Releases](doc:releases) section for more details on releases|
-|A library to be used in other systems|lib or app|use 'lib' for stateless libraries that contain modules, and 'app' for stateful libraries with a supervision tree|
-|A collection of multiple libraries|umbrella|This is the one form of project supported where multiple top-level apps are used.
-
-These projects cannot be used as dependencies.
-
-For projects usable as dependencies, see [Vendoring Plugins](https://www.rebar3.org/v3/docs/using-available-plugins#section-vendoring-dependencies)|
+|A short script or util|escript|The user you distribute this to will need to have Erlang installed.<br><br>Dependencies in C are not trivially inclduded nor redistributed|
```
Comment (tsloughter) [inline content/en/docs/workflow.md:15]:
```suggestion
|A short script or util|escript|The user you distribute this to will need to have Erlang installed.<br><br>Dependencies in C are not trivially included nor redistributed|
```

File: content/en/docs/workflow.md [hunk 3]
```diff
@@ -1,49 +1,41 @@
...
+|A full, self-contained, executable system|release or umbrella|This is the recommended production deploy for Erlang systems.<br><br>See the [Releases](/docs/releases) section for more details on releases|
+|A library to be used in other systems|lib or app|use `lib` for stateless libraries that contain modules, and `app` for stateful libraries with a supervision tree|
+|A collection of multiple libraries|umbrella|This is the one form of project supported where multiple top-level apps are used.<br><br>These projects cannot be used as dependencies.<br>For projects usable as dependencies, see [Vendoring Plugins](https://www.rebar3.org/v3/docs/using-available-plugins#section-vendoring-dependencies)|
```
Comment (tsloughter) [inline content/en/docs/workflow.md:18]: Why does this link to the vendoing project to talk about "projects usable as dependencies"?

File: content/en/docs/getting-started.md (hunk 1 above)
Comment (ferd) [reply content/en/docs/getting-started.md:7]: Yeah. Funnily enough I reused the AdoptingErlang blurb, but I can just swap it for "wraps" or "integrates"

File: content/en/docs/workflow.md (hunk 3 above)
Comment (ferd) [reply content/en/docs/workflow.md:18]: This is in the context of an umbrella applications when that was written on the site.. I should probably change it and mention git_subdir deps instead now.

**Pull request merged**